### PR TITLE
Sed stable

### PR DESCRIPTION
**Pull request by william-gr**

### CODE REVIEW

File: src/freenas/etc/ix.rc.d/ix-sed [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
```
[the remark this answers is not in the document]
Comment (william-gr) [reply src/freenas/etc/ix.rc.d/ix-sed]: @themylogin i dont follow : p you wanted bash?

[the remark this answers is not in the document]
Comment (themylogin) [reply src/freenas/etc/ix.rc.d/ix-sed]: @william-gr you've talked me out of it!

**Comment by amotin** [comment]
I see two issues with this patch:
 -- The biggest is that it uses different technology -- ATA Security (camcontrol security) instead of Trusted Computing Group OPAL/Enterprise SSC (sedutil).  ATA drives supporting Opal may also support ATA Security, but those are not equivalent and can not be used together (enabling one blocks another).  Plus SAS disks do not have alternative to ATA Security and should use TCG Enterprise SSC APIs. At least the rc.d script has to be improved to support `sedutil`, possibly in addition to `camcontrol security`.
 -- I see there code only to unlock devices once on boot.  But what is about hot-plug?  I suppose we'll have to track some devd events.

**Comment by william-gr** [comment]
@amotin this is just merging code already in master sent by Josh. You OK if we handle the issues in another ticket/PR?

**Comment by amotin** [comment]
Not that I am against that, the patch probably does what it is supposed to, but as I have told it is different technology.  It even in UI explicitly mentions `camcontrol security`, that is probably not opaque enough for universal approach as I see it now.

**Comment by amotin** [comment]
Sure. Sync with master is good. I just commented on PR you gave me.

**Pull request merged**